lib/bot.js

```diff
@@ -154,7 +154,8 @@ class Bot {
     if (author.id === this.discord.user.id) return;
 
     const channelName = `#${message.channel.name}`;
-    const ircChannel = this.channelMapping[channelName];
+    const ircChannel = this.channelMapping[message.channel.id in this.channelMapping ?
```

Comment:
`const ircChannel = this.channelMapping[message.channel.id] || this.channelMapping[channelName]` maybe?